packages/faucet/src/types.ts

```diff
@@ -1,7 +1,32 @@
-import { Address, Amount, Identity } from "@iov/bcp";
+import { Account, Coin } from "@cosmwasm/sdk38";
 
 export interface SendJob {
-  readonly sender: Identity;
-  readonly recipient: Address;
-  readonly amount: Amount;
+  readonly sender: string;
+  readonly recipient: string;
+  readonly amount: Coin;
 }
+
+export interface BankTokenMeta {
+  readonly denom: string;
+  /**
+   * The token ticker symbol, e.g. ATOM or ETH.
+   */
+  readonly tickerSymbol: string;
+  /**
+   * The number of fractional digits the token supports.
+   *
+   * A quantity is expressed as atomic units. 10^fractionalDigits of those
+   * atomic units make up 1 token.
+   *
+   * E.g. in Ethereum 10^18 wei are 1 ETH and from the quantity 123000000000000000000
+   * the last 18 digits are the fractional part and the rest the wole part.
+   */
+  readonly fractionalDigits: number;
```

Comment:
Do we want this still? Or get rid of it for now?
We can have a working faucet that uses native tokens.